README.md

```diff
@@ -1 +1,331 @@
+
+--->
+
+## 👥 Authors
+
+👤 *Jonathan Kiprono*
+- GitHub:(https://github.com/Jonathan-kiprono)
+- LinkedIn: [LinkedIn][(https://www.linkedin.com/in/jonathan-kiprono-8858a9193/overlay/contact-info/)]
+
+## 🔭 Future Features
+
+- [ ] *[Add more forms]*
+
+
+
+## 🔭 Future Features
+
+- [ ] *[Add more forms]*
+- [ ] *[Add more pages with information]*
+- [ ] *[Add some Javascript]*
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support
+
+If you like this project feel free to share with your friends.
+
+## 🙏 Acknowledgments
+
+I would like to thank Microverse for this amazing exercise.
+
+## 📝 License
+
+This project is [MIT](https://github.com/ntimanu/portfolio-project/blob/htmlcssbranch/LICENCE) licensed.
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+# 📖 [Portfolio: setup and mobile first]
+
+*[Portfolio: setup and mobile first]* is a simple project that show a page of a portfolio that discribe the work, experience and contacts of the portfolio owner.
+
+## 🛠 Built With
+
+### Tech Stack
+HTML
+CSS
+JS
+Git/GitHub
+node.js
+Boostrap 5
+
+### Key Features
+
+- *[A webpage that gives you all the information of the owner]*
+
+## 🚀 Live Demo
+
+- [Comming Soon]
+
+## 💻 Getting Started
+
+When you click on the page, the browser shows you the information of the owner.
+
+### Prerequisites
+
+Clone this repository to your desired folder and open html file.
+
+### Setup
+
+Clone this repository to your desired folder:
+Example commands:
+
+sh
+  cd portfolio
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+sh
+  cd potfolio
+  npm install
+
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+sh
+  open live server
+
+--->
+
+## 👥 Authors
+
+👤 *Jonathan Kiorono*
+
+- GitHub: (https://github.com/Jonathan-kiprono)
+- LinkedIn: [LinkedIn][(https://www.linkedin.com/in/jonathan-kiprono-8858a9193/overlay/contact-info/)]
+
+## 🔭 Future Features
+
+- [ ] *[Add more forms]*
+- [ ] *[Add more pages with information]*
+- [ ] *[Add some Javascript]*
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support
+
+If you like this project feel free to share with your friends.
+
+## 🙏 Acknowledgments
+
+I would like to thank Microverse for this amazing exercise.
+
+## 📝 License
+
+This project is [MIT](https://github.com/ntimanu/portfolio-project/blob/htmlcssbranch/LICENCE) licensed.
```

Comment:
- [ ] I think this is some mistake. You have 3 versions of your README file in one document. Apart from the fact that professional developers must avoid unnecessary repetition in their code,even in the documentation, one must give utmost attention to it because this is the first contact for anyone who visits your repository, and you want to show how careful and detailed you are with your style of documentation. Kindly delete the duplicates please. 🙏